engine/combat/utility/sos_appraisal.py

```diff
@@ -133,7 +133,8 @@ def execute_confirm_command(self: Self) -> None:
 
     def execute_selecting_skill(self: Self) -> None:
         if (
```

Comment:
Logic for the condition here may be getting complex enough to pull it out into a variable and give it a name, in this case `skill_not_selected`?

Reply:
I will do this during the controller refactor